static/src/javascripts/projects/common/modules/commercial/comment-adverts.js

```diff
@@ -44,6 +44,7 @@ define([
             !$adSlotContainer.length ||
             !config.switches.discussion ||
             !identityApi.isUserLoggedIn() ||
+            (config.page.section === 'childrens-books-site' || config.page.shouldHideAdverts) ||
```

Comment:
This not the same as 2 separate `or` statements?

`a || (b || c) || d == a || b || c || d`